tests/scripts/test_stdlib.py

```diff
@@ -24,6 +24,20 @@ def test_check_multiline_output_no_split():
     assert run(a_command, split=False)['stdout'] == u'This a multi-\nline No Split test!\n'
 
 
+def test_check_error():
+    a_command = ['false']
```

Comment:
I'd prefer not 'false' but something self-speaking like 'nosuchcommand'

Reply:
I think we should be testing exit codes of existing commands here.

Reply:
makes sense